expression/expression.go

```diff
@@ -311,8 +336,6 @@ func VecEvalBool(ctx sessionctx.Context, exprList CNFExprs, input *chunk.Chunk,
 
 func vecEval(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, result *chunk.Column) (err error) {
 	switch expr.GetType().EvalType() {
-	case types.ETInt:
```

Comment:
Maybe we can maintain this, and call vecEval in line 252, and then check `if eType == types.ETInt {`